modules/account-lib/src/coin/algo/keyPair.ts

```diff
@@ -19,12 +31,20 @@ export class KeyPair extends Ed25519KeyPair {
   }
 
   /** @inheritdoc */
-  getAddress(format: AddressFormat): string {
-    throw new NotImplementedError('getAddress not implemented');
+  getAddress(format: AddressFormat = AddressFormat.string): string {
```

Comment:
why are we changing the specification of the base class here? Is there a reason why we cant return a `hex` or `base58` encoding of the string?